fileio/private/read_biosemi_bdf.m

```diff
@@ -254,20 +254,52 @@
   endsample = endsample - (begepoch-1)*epochlength;  % correct for the number of bytes that were skipped
   dat = dat(:, begsample:endsample);
 
-  % Calibrate the data
-  calib = diag(EDF.Cal(chanindx));
+% -------------------------------------------------------------------------
+% data calibration
+% ----------------
+% The values of the selected channels are multiplied with a calibration
+% coefficient which is specified by (EDF.PhysMax - EDF.PhysMin)/(EDF.DigMax
+% - EDF.DigMin).
+% -------------------------------------------------------------------------
+  calib = diag(EDF.Cal(chanindx)); % get the calibration coefficients
+  
+  % Change the calibration coefficient of the 'Status' channel to 1
+  %
+  % Biosemi in general puts also for the status channel the eeg 
+  % calibration parameters to the header. A calibration of the 'Status'
+  % channel with the eeg calibration parameters results in nonsense.
+  % Therefore it will be corrected here.
+  chanlabels = cellstr(EDF.Label); 
+  labelindx = find(strncmp(chanlabels, 'Status', 6)); % locate 'Status' label
```

Comment:
this should be a case-insensitive comparison, i.e. use strcmpi rather than strncmp